src/main/java/org/alliancegenome/curation_api/view/View.java

```diff
@@ -5,7 +5,7 @@
     public static class FieldsOnly { }
     public static class FieldsAndLists extends FieldsOnly { }
     
-    public static class VocabularyTermView extends FieldsOnly { }
+    public static class VocabularyTermView extends FieldsAndLists { }
```

Comment:
@markquintontulloch this might cause stackover flow is some cases where Vocab -> Terms and Terms -> Vocab

Reply:
Nevermind:

	@JsonView({View.VocabularyView.class})
	private List<VocabularyTerm> memberTerms;
	
	@JsonView({View.VocabularyTermView.class})
	private Vocabulary vocabulary;
	
I think you are fine.